libcontainer/factory_nabla.go

```diff
@@ -91,10 +91,13 @@ func createRootfsISO(config *configs.Config, containerRoot string) (string, erro
 		if (mount.Destination == "/etc/resolv.conf") ||
 			(mount.Destination == "/etc/hosts") ||
 			(mount.Destination == "/etc/hostname") {
+			if err := os.MkdirAll(filepath.Join(rootfsPath+"/etc"), 0755); err != nil {
+				return "", errors.Wrap(err, "Unable to create "+rootfsPath+"/etc")
+			}
```

Comment:
Could we move this out of the loop? If not, LGTM. Thanks for the contribution!

Reply:
Absolutely